packages/file-storage/.env.test.sample

```diff
@@ -0,0 +1,3 @@
+ACCESS_KEY_ID=
+BUCKET=
+ENDPOINT=
```

Comment:
in endpoint what's the expected pattern ? only the AWS region ?

Reply:
It should be `https://s3.eu-central-1.amazonaws.com/`, I'll update the example